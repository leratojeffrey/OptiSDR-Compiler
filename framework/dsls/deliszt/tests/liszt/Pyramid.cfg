{
  "main-class": "Pyramid",
  "mesh-file": "../../../Pyramid.vtu",
  "runtimes": ["single"],
  "log": ["runtime","ASTUtil"],
  "floating-point-type": "float"
}