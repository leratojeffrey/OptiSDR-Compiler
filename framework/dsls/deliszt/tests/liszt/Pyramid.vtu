<VTKFile type="UnstructuredGrid" version="0.1" byte_order="BigEndian">
  <UnstructuredGrid>
    <Piece NumberOfPoints="5" NumberOfCells="1">
      <PointData Scalars="scalars">
      </PointData>
      <CellData Scalars="scalars">
      </CellData>
      <Points>
        <DataArray type="Float32" NumberOfComponents="3" Format="ascii">
         0.0 0.0 0.0
         1.0 0.0 0.0
         1.0 0.0 -1.0
         0.0 0.0 -1.0
         0.5 1.0 -0.5
        </DataArray>
      </Points>
      <Cells>
        <DataArray type="Int32" Name="connectivity" Format="ascii">
          0 1 2 3 4
        </DataArray>
        <DataArray type="Int32" Name="offsets" Format="ascii">
          5
        </DataArray>
        <DataArray type="Int32" Name="types" Format="ascii">
          14
        </DataArray>
      </Cells>
    </Piece>
  </UnstructuredGrid>
</VTKFile>