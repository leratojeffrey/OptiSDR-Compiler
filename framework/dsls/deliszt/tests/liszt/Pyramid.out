liszt: Cell 1: vVertex 0 0
liszt: Cell 1: vVertex 1 1
liszt: Cell 1: vVertex 2 2
liszt: Cell 1: vVertex 3 3
liszt: Cell 1: vVertex 4 4
liszt: Cell 1: eEdge 3 0
liszt: Cell 1: eEdge 0 1
liszt: Cell 1: eEdge 5 2
liszt: Cell 1: eEdge 2 3
liszt: Cell 1: eEdge 4 4
liszt: Cell 1: eEdge 1 5
liszt: Cell 1: eEdge 6 6
liszt: Cell 1: eEdge 7 7
liszt: Cell 1: fFace 0 0
liszt: Cell 1: fFace 4 1
liszt: Cell 1: fFace 1 2
liszt: Cell 1: fFace 2 3
liszt: Cell 1: fFace 3 4
